ansible/roles/atmo-vnc/tasks/realvnc.yml

```diff
@@ -100,6 +100,15 @@
     owner: "{{ ATMOUSERNAME }}"
   when: (ansible_distribution == "CentOS") and (ansible_distribution_major_version == '7')
```

Comment:
This line can be modified to end with "...ansible_distribution_major_version|int >= 7" . If you want to make that change, I'll accept. or I can make the change if you'd like.

While minor, this would save an extra check in ansible.

The "name" of this task should also be edited to reflect centos 7 & 8.